check_stability.py

```diff
@@ -353,6 +353,53 @@ def get_files_changed():
             for item in files[:-1].split("\0")]
 
 
+def get_affected_testfiles(files_changed):
+    affected_testfiles = []
+    ignore_dirs = ["conformance-checkers", "docs"]
+    manifest_file = os.path.join(os.path.abspath(os.curdir), "w3c", "web-platform-tests", "MANIFEST.json")
+    with open(manifest_file, "r") as fh:
+        manifest_contents = fh.read()
+    for changedfile_pathname in files_changed:
+        repo_root = os.path.abspath(os.path.join(os.path.abspath(os.curdir), "w3c", "web-platform-tests"))
+        changed_file_repo_path = changedfile_pathname[len(repo_root):]
```

Comment:
> This could probably use `os.path.relpath`

Yeah I thought the same when I wrote it

> but it's not critical

Maybe not, but worth doing right anyway :) So, changed it now to use `os.path.relpath`